src/lib/y2network/sequences/interface.rb

```diff
@@ -2,7 +2,11 @@
 
 module Y2Network
   module Sequences
-    # Sequence for interface
+    # The response of this class is to drive workflow for sequence of dialogs.
```

Comment:
I guess you mean "responsability", right?

Reply:
yep